### ISSUE TITLE
Having a doi for each version of each workflow

### ISSUE THREAD

**Issue by lldelisle** [galaxyproject/iwc#287]
We are facing an issue between orcid-workflowhub-zenodo.
It seems that a way to do it would be to follow this:
https://github.com/zenodo/zenodo/issues/1814#issuecomment-620555444
To be able to link all the repos of iwc-workflows to zenodo. This way each release will generate a new doi.

**Comment by mvdbeek** [comment]
ok, i've done that now

**Comment by mvdbeek** [comment]
I think this is now the case, we get DOIs for everything on dockstore. We should pull them into the website as well

**Comment by lldelisle** [comment]
There are workflows where I don't see them in dockstore:
- https://dockstore.org/workflows/github.com/iwc-workflows/cutandrun/main:main?tab=info
- https://dockstore.org/workflows/github.com/iwc-workflows/atacseq/main:main?tab=info
And I see it for:
- https://dockstore.org/workflows/github.com/iwc-workflows/chipseq-pe/main:main?tab=info

**Comment by dannon** [comment]
I looked into this last week to include on the site - I think they may only be adding them with new updates, so some older workflows won't have them yet

**Comment by mvdbeek** [comment]
Yep, if you want to integrate it I can make sure everything has a doi.
